app/controllers/admin/schools/cohorts/change_programme_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Admin
+  module Schools
+    module Cohorts
+      class ChangeProgrammeController < ::Admin::BaseController
+        skip_after_action :verify_authorized
+        skip_after_action :verify_policy_scoped
```

Comment:
Just a general moan about this.

Reply:
Yeah, understood. I wanted to bake a load of the logic about what was allowed and not into the policy, not the service, but couldn't find a nice way to do it, since it depends on previous and current state